Jenkinsfile

```diff
@@ -28,6 +30,8 @@ properties([
     ])
 ])
 
+def s3_builddir = "fcos-builds/prod/streams/${params.STREAM}"
```

Comment:
where is this path coming from? did we define our s3 bucket layout anywhere?

Reply:
That's https://github.com/coreos/fedora-coreos-tracker/issues/189, right?

Reply:
I added a comment with the link. We can definitely still move builds around after this merges if discussions in that ticket lead to a different layout.

Reply:
> That's [coreos/fedora-coreos-tracker#189](https://github.com/coreos/fedora-coreos-tracker/issues/189), right?

right. I'm actually OK with the layout (as you mentioned being defined over in that ticket. I think we should make the bucket name a parameter. We may be using `fcos-builds` but someone without access to that bucket may want to try to setup a pipeline and we should make it possible.

Reply:
I added a patch to #62 instead to fix this because this intersects with the devel path and it would've been messy to understand the changes if they were split across here and there.